src/test/java/com/google/api/codegen/testdata/python_no_path_templates.baseline

```diff
@@ -132,6 +132,11 @@ class NoTemplatesServiceApi(object):
         and trailing
         whitespace.
 
+        Example:
+          >>> from google.example.v1 import no_path_templates_pb2
+          >>> api = NoTemplatesServiceApi()
```

Comment:
Hmm... good catch. Anyone know why PythonImportHandler is not importing the class name (i.e. `from google.example.v1.no_path_templates_pb2 import NoTemplatesServiceApi`)?

The handwritten Python snippets for PubSub suggest that approach instead: http://pythonhosted.org/gax-google-pubsub-v1/google.pubsub.v1.publisher_api.html